tests/conftest.py

```diff
@@ -9,12 +9,12 @@ def test_env():
     os.environ = {
         "ALMA_API_URL": "test",
         "ALMA_API_READ_WRITE_KEY": "test",
-        "LOG_LEVEL": "test",
+        "LOG_LEVEL": "DEBUG",
         "SAP_DROPBOX_CONNECTION": "test",
         "SAP_REPLY_TO_EMAIL": "test",
         "SAP_FINAL_RECIPIENT_EMAIL": "test",
         "SAP_REVIEW_RECIPIENT_EMAIL": "test",
-        "SENTRY_DSN": "test",
+        "SENTRY_DSN": None,
```

Comment:
Quoting Helen, since I did this too, this should be `"None"` since that's what it will be when read from the Dev1 env.

Reply:
I can't tell which value you are referring to. I thought these would supercede what's in the dev1 env... Isn't that the reason for making the fixture?

Reply:
`SENTRY_DSN`, it's just a matter of making the fixture more like the real thing

Reply:
It doesn't really matter what it is here, we just need to have a test for both cases (`None` and `"None"`).